include/glow/Backends/CompiledFunction.h

```diff
@@ -16,12 +16,47 @@
 #ifndef GLOW_BACKENDS_COMPILEDFUNCTION_H
 #define GLOW_BACKENDS_COMPILEDFUNCTION_H
 
+#include "glow/Graph/Nodes.h"
 #include <unordered_map>
 
 namespace glow {
 
 class Context;
+namespace runtime {
+/// Runtime Symbol Info
```

Comment:
Nit: I fell this should be the class comment and need to be more explanatory.
http://llvm.org/docs/CodingStandards.html#commenting